modules/test/playwright/pages/portal-workflow-kaleo-designer-web/ProcessBuilderPage.ts

```diff
@@ -6,22 +6,29 @@
 import {Page} from '@playwright/test';
 
 import {ApplicationsMenuPage} from '../product-navigation-applications-menu/ApplicationsMenuPage';
+import {ConfigurationTabPage} from './ConfigurationTabPage';
 import {DiagramViewPage} from './DiagramViewPage';
 import {SourceViewPage} from './SourceViewPage';
 
 export class ProcessBuilderPage {
 	readonly applicationsMenuPage: ApplicationsMenuPage;
 	readonly diagramViewPage: DiagramViewPage;
+	readonly configurationTabPage: ConfigurationTabPage;
 	readonly page: Page;
 	readonly sourceViewPage: SourceViewPage;
 
 	constructor(page: Page) {
 		this.applicationsMenuPage = new ApplicationsMenuPage(page);
 		this.diagramViewPage = new DiagramViewPage(page);
+		this.configurationTabPage = new ConfigurationTabPage(page);
 		this.page = page;
 		this.sourceViewPage = new SourceViewPage(page);
 	}
 
+	async goToConfigurationTab() {
+		await this.configurationTabPage.goTo();
+	}
```

Comment:
Hey Carlos, I couldn't understand why we have a function just calling another function?

We could not directly call `configurationTabPage.goTo()` directly?

Reply:
You can call a function to go to the configuration tab either from one page or another, it is just a matter os convenience, but I see your point and have deleted.